### PR TITLE
[PD-7468] - radius search input

### PR DESCRIPTION
**Pull request by surgiie**
## What does this PR do?

* Reworks radius search code.
* Remove old search form
* Suggest a "Use my current location" if the user hasnt explicitly blocked geolocation permissions when previously prompted.

### CODE REVIEW

File: src/components/Form/AppAutocompleteInput.vue [hunk 1]
```diff
@@ -14,25 +14,33 @@
         >
             {{ label }}
         </label>
+        <div :class="hasAddOns ? 'form__input-group': ''">
```
Comment (jeremytubbs) [inline src/components/Form/AppAutocompleteInput.vue:17]: change to `:class="{'form__input-group': hasAddOns}"`

File: src/components/Form/AppAutocompleteInput.vue [hunk 2]
```diff
@@ -132,6 +149,7 @@ export default {
         },
         blur(event) {
             this.$emit("input", event.target.value)
+            this.$emit('blur', event.target.value)
```
Comment (jeremytubbs) [inline src/components/Form/AppAutocompleteInput.vue:152]: Need to run a formatter.

`this.$emit("blur", event.target.value)`

File: src/demo/components/SearchForm.vue [hunk 3]
```diff
@@ -21,17 +21,62 @@
...
-                    label="Location"
+                    label="Where"
                     ref="location"
-                    v-model="formData.location"
-                    placeholder="city, state, country or zip code"
-                    aria-label="Search by location"
-                    aria-labelledby="form__label-location"
-                    autocapitalize="off"
                     autocomplete="off"
+                    @setResult="search"
```
Comment (jeremytubbs) [inline src/demo/components/SearchForm.vue:27]: please remove `@setResult` the user doesn't get a chance to select the radius

**Review by jeremytubbs — CHANGES_REQUESTED** [review]
Minor changes

File: src/components/Form/AppAutocompleteInput.vue [hunk 4]
```diff
@@ -14,25 +14,35 @@
...
-            @keydown.esc="blur"
-            @keydown.up="keyUp"
-            @keydown.down="keyDown"
-            aria-autocomplete="list"
-            aria-haspopup="listbox"
-            :aria-labelledby="`form__label-${id}`"
-            :aria-activedescendant="activeDescendant"
-        />
+            <input
+                ref="input"
```
Comment (jeremytubbs) [inline src/components/Form/AppAutocompleteInput.vue:25]: I don't like sorting things by length.. I think keeping them grouped is better.

**Review by jeremytubbs — APPROVED** [review]

**Review by nate998877 — APPROVED** [review]

**Pull request merged**